### ISSUE TITLE
Should "Hardware Accelerated GPU Scheduling" be turned On or OFF? 

### ISSUE THREAD

**Issue by DeemChan** [NVIDIA/Stable-Diffusion-WebUI-TensorRT#107]
Should "Hardware Accelerated GPU Scheduling" be turned On or OFF? (on Windows based machine)

![image](https://github.com/NVIDIA/Stable-Diffusion-WebUI-TensorRT/assets/15304538/534bcf3f-6a8e-4e20-8c31-ef6bf8a897ea)

I heard different opinions on how turning it ON or OFF affects the performance.
Does it even do? 
If yes, what would be the optimal "position" of the switch? On or Off?

**Comment by MorkTheOrk** [comment]
You can try both, it depends on what you use.

**Comment by Zaazu** [comment]
I get faster iterations with GPU scheduling disabled

**Comment by contentis** [comment]
Turning it off is almost always better. The amount of acceleration you'll see heavily depends on the model and GPU and can range anywhere from <1% to 10%

**Comment by DeemChan** [comment]
Turning it OFF then!
Thank you for sharing guys!

**Comment by BurnZeZ** [comment]
It would be nice to see some actual comparisons, if anyone is up to it.

**Comment by J-Cott** [comment]
I tried turning if off. But it actually made the generation slightly slower for me, by 1.1 Seconds for 4 images, (25 seconds vs 23.9 seconds with it turned on). I have a RTX 3090 and 11900K with 64 GB Ram. And use TensorRT.

**Comment by Genzo9319** [comment]
I did a little testing

SD generation details:
SD arguments: --opt-sdp-attention --lowvram --xformers
SD Settings: No previews
CKPT-epicrealism
Txt2Img; prompt: cat
Sampler: DPM++ 2M Karras
Size 512x512
Steps: 20
Batch count: 2
CFG: 7

Results:
Without Hardware Acceleration:  1 image 48s, 2.42s/it, total time: 1:37s
With Hardware Acceleration: 1 image 45s, 2.30s/it, total time: 1:32s

Conclusion:
When Hardware Accelerated GPU Scheduling was turned on, image generation was faster.

Laptop:
OS - Windows 10 Pro N (19045) 22H2 Media Feature Pack for Windows 10/11 N (February 2023)
GPU - Nvidia GTX 1050 ti - 4GB VRAM, Performance settings
CPU - Intel Core i7 8750H @ 2.20GHz
SSD - XPG GAMMIX S70 BLADE (SSD) 1 TB
RAM - Samsung DDR4-2666 (1333 MHz) 16 Gb
Browser: Google chrome
Laptop was restarted before each generation batch. Upon restart, that was the first thing I did.
